src/test/java/microsoft/exchange/webservices/data/core/EwsUtilitiesTest.java

```diff
@@ -54,6 +56,11 @@ public void testGetBuildVersion() {
   }
 
   @Test
+  public void testParseInt() throws IllegalAccessException, ParseException, InstantiationException {
+    Assert.assertEquals(new Integer(14), EwsUtilities.parse(Integer.class, "14"));
+  }
```

Comment:
thanks. I just thought about having this fixed if we are already at this point. The unit-test wasn´t there before I just created it to underline the defect. If you don't want to fix this here please create an issue for it.